packages/version-select/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "@hashicorp/version-select",
+  "description": "A component to handle selecting a version of docs to navigate to.",
+  "version": "0.1.0",
```

Comment:
Might want to start this at `0.0.0` so that it kicks off its life with the first increment!

Reply:
updated to `0.0.0`!